config/spec/mainnet.go

```diff
@@ -151,10 +150,13 @@ func MainnetChainSpecData() *chain.SpecData {
 		// Deneb1 values.
 		EVMInflationAddressDeneb1:  common.NewExecutionAddressFromHex(mainnetEVMInflationAddressDeneb1),
 		EVMInflationPerBlockDeneb1: mainnetEVMInflationPerBlockDeneb1,
+
+		// Electra values.
+		MinValidatorWithdrawabilityDelay: defaultMinValidatorWithdrawabilityDelay,
```

Comment:
TODO: determine correct value here. As @rezbera noted: 256 epochs at 384 seconds per epoch is ~27 hours.